defaults/main.yml

```diff
@@ -41,7 +41,7 @@ pgbouncer_stats_users:
 # APT settings
 pgbouncer_apt_key_id: "ACCC4CF8"
 pgbouncer_apt_key_url: "https://www.postgresql.org/media/keys/ACCC4CF8.asc"
-pgbouncer_apt_repository: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main"
+pgbouncer_apt_repository: "deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release }}-pgdg main {{ postgresql_version | default ('') }}"
```

Comment:
This doesn't look right.

`postgresql_version` is an external variable, which this role should not depend on.

Instead of making changes in this role, I think, it is better to fix PosgresSQL role, and remove the version suffix from `postgresql_apt_repository` there.

Reply:
This makes sense, I was trying to keep the postgresql role close to the forked repo, but it make more sense to adjust there,  I will update and retest

Reply:
This fix has been moved to https://github.com/miarec/ansible-postgresql/pull/4 and re-tested